haystack/tracing/datadog.py

```diff
@@ -11,10 +11,12 @@
 
 with LazyImport("Run 'pip install ddtrace'") as ddtrace_import:
     import ddtrace
+    from ddtrace._trace.span import Span as ddSpan
+    from ddtrace._trace.tracer import Tracer as ddTracer
```

Comment:
due to https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/trace/__init__.py this should also work
```suggestion
    from ddtrace.trace import Span as ddSpan
    from ddtrace.trace import Tracer as ddTracer
```